### PR TITLE
Add initial README and crate-level docs

### PR DESCRIPTION
**Pull request by tnull**

### CODE REVIEW

**Comment by moneyball** [comment]
Looks like a good initial write-up to me.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
 # LDK Node
-A ready-to-go node implementation built using LDK.
+A ready-to-go Lightning node library built using [LDK](https://lightningdevkit.org/) and [BDK](https://bitcoindevkit.org/).
+
+LDK Node is a non-custodial Lightning node in library form. Its central goal is to provide a small, simple, and straightforward interface that enables users to easily setup and run a Lightning node with an integrated on-chain wallet. While minimalism is at its core, LDK Node aims to be sufficiently modular and configurable to be useful for a variety of use cases.
```
Comment (jkczyz) [inline README.md:4]: @moneyball Any opinion on self-custodial vs non-custodial?

Comment (jkczyz) [inline README.md:4]: s/setup/set up

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
 # LDK Node
-A ready-to-go node implementation built using LDK.
+A ready-to-go Lightning node library built using [LDK](https://lightningdevkit.org/) and [BDK](https://bitcoindevkit.org/).
+
+LDK Node is a non-custodial Lightning node in library form. Its central goal is to provide a small, simple, and straightforward interface that enables users to easily setup and run a Lightning node with an integrated on-chain wallet. While minimalism is at its core, LDK Node aims to be sufficiently modular and configurable to be useful for a variety of use cases. 
+
+## Getting Started
+
+The main interface of the library is the `Node`, which can be retrieved by setting up and configuring a `Builder` to the user's liking and calling `build()`. `Node` can then be controlled via commands such as `start`,`stop`,`connect_open_channel`, `send_payment`, etc.:
```
Comment (jkczyz) [inline README.md:8]: Would recommend "your" over "user's".

File: src/lib.rs [hunk 3]
```diff
@@ -482,7 +520,7 @@ struct Runtime {
 	stop_wallet_sync: Arc<AtomicBool>,
 }
 
-/// The main interface object of the simplified API, wrapping the necessary LDK and BDK functionalities.
+/// The main interface object of LDK Node, wrapping the necessary LDK and BDK functionalities.
```
Comment (jkczyz) [inline src/lib.rs:523]: Would consider "primary abstraction" instead of "main interface object".

**Review by jkczyz — COMMENTED** [review]
Largely looks good.

File: README.md (hunk 1 above)
Comment (moneyball) [reply README.md]: I don't feel strongly about it. Non-custodial is fine.

**Review by jkczyz — APPROVED** [review]
Feel free to squash

**Pull request merged**